### DISCUSSION TITLE
Paste docs site navigation feedback

### DISCUSSION THREAD

**Discussion by dprothero** [twilio-labs/paste#2346] [Ideas]
The Paste docs site is fantastic! One issue, however, is the left nav doesn't stay pinned to where you had it scrolled after clicking on a new item. It scrolls back to the top.

**Comment by TheSisb** [comment]
Hey @dprothero,

Thanks for that feedback and I agree it can be annoying to lose the scroll position while clicking through the navigation.  I created a ticket to address this: https://issues.corp.twilio.com/browse/DSYS-3308.  We'll report back after it gets fixed.
